code/provision/playbooks/roles/ingress-controller/templates/values.yml.j2

```diff
@@ -5,7 +5,7 @@ controller:
   kind: Deployment
   extraArgs:
     default-ssl-certificate: "kube-system/tls-secret"
-    annotations-prefix: "ingress.kubernetes.io"
+    annotations-prefix: "nginx.ingress.kubernetes.io"
```

Comment:
Do we need to provide this argument if we're using the default?

Reply:
Yes you're right - removed.